deploy/manifests/dev-v2/us-east-2/cluster/storetheindex/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: storetheindex
+
+commonLabels:
+  toolkit.fluxcd.io/tenant: storetheindex
+
+resources:
+  - namespace.yaml
+  - flux-cd.yaml
+  # - flux-rbac.yaml
```

Comment:
Ditto re documenting rationale.

Reply:
I need to develop the tenant level restrictions. This flag is not required currently because I need complete the restrictive flow and it's currently disabled.
https://github.com/filecoin-project/gitops-root/issues/191

You can see that the flow is currently restricted here.
https://github.com/filecoin-project/gitops-root/blob/a71686bd1dc8837f0beac8b7af21e78aefa8e821/.weave-gitops/clusters/dev-v2/system/flux-system/kustomization.yaml#L13

I should complete this today/tomorrow.